mastermsm/trajectory/traj.py

```diff
@@ -24,6 +25,58 @@ def _load_mdtraj(top=None, traj=None):
     """
     return md.load(traj, top=top)
 
+class MultiTimeSeries(object):
+    """ A class for generating multiple TimeSeries objects in
+    a consistent way. In principle this is only needed when
+    the clustering is not established a priori.
+
+    """
+    def __init__(self, top=None, trajs=None, dt=None):
+        """
+        Parameters
+        ----------
+        dt : float
+            The time step.
+        top : string
+            The topology file, may be a PDB or GRO file.
+        trajs : list 
+            A list of trajectory filenames to be read.
+
+        """
+        self.file_list = trajs
+        self.traj_list = []
+        for traj in self.file_list:
+            tr = TimeSeries(top=top, traj=traj)
+            self.traj_list.append(tr)
+    
+    def joint_discretize(self):
+        """
+        Analyze jointly torsion angles from all trajectories.
+
+        Produces a fake trajectory comprising a concatenated set
+        to recover the labels from HDBSCAN.
+
+        """
+        phi_cum = []
+        psi_cum = []
+        for tr in self.traj_list:
+            phi = md.compute_phi(tr.mdt)
+            psi = md.compute_psi(tr.mdt)
+            phi_cum.append(phi[1])
+            psi_cum.append(psi[1])
+        phi_cum = np.vstack(phi_cum)
+        psi_cum = np.vstack(psi_cum)  
+        phi_fake = [phi[0], phi_cum]
```

Comment:
There is a mistake here, it should be phi_fake = [phi[1], phi_cum], since phi[1] (and not phi[0]) has the length of number of frames in one of the parallel trajectories.
Obviously, the same applies to psi_fake.